recipe/meta.yaml

```
@@ -15,13 +15,14 @@ source:
 build:
     number: 1004
     script: python setup.py install --single-version-externally-managed --record record.txt
```

```suggestion
    script: python -m pip install . -vv
```

Other options are now the default.